Ocelot.Downstream.Service.Instance2/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,39 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace Ocelot.Downstream.Service.Instance2.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class WeatherForecastController : ControllerBase
+    {
+        private static readonly string[] Summaries = new[]
+        {
+        "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
+    };
+
+        private readonly ILogger<WeatherForecastController> _logger;
+
+        public WeatherForecastController(ILogger<WeatherForecastController> logger)
+        {
+            _logger = logger;
+        }
+
+        [HttpGet("hostname")]
+        public string GetHostName()
```

Comment:
Inject IOptions with custom properties, such as "InstanceName" : "Instance1"/"Instance2" in constructor